client/src/Components/HomePage/header.css

```diff
@@ -0,0 +1,38 @@
+/* img {
+	width: 48px;
+	height: 48px;
+	margin: 20px;
+} */
+
+.parentDiv {
+	display: flex;
+	justify-content: space-between;
+	border-bottom: 1px solid black;
```

Comment:
color of the border is not black in design

Reply:
border part is done